content/200-orm/050-overview/500-databases/300-postgresql.mdx

```diff
@@ -91,7 +91,7 @@ Notice that this code requires the `DATABASE_URL` environment variable to be set
 
 ### Connection URL
 
-Prisma is based on the [official PostgreSQL format](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING) for connection URLs, but does not support all arguments and includes additional arguments such as `schema`. Here's an overview of the components needed for a PostgreSQL connection URL:
+Prisma ORM is based on the [official PostgreSQL format](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING) for connection URLs, but does not support all arguments and includes additional arguments such as `schema`. Here's an overview of the components needed for a PostgreSQL connection URL:
```

Comment:
Unrelated to the PR:
This sentence does not really make sense. "Prisma ORM is based on ...". Is it missing "'s connection string" or something like that?

Reply:
```suggestion
Prisma ORM follows the connection URL format specified by [PostgreSQL's official guidelines](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING), but does not support all arguments and includes additional arguments such as `schema`. Here's an overview of the components needed for a PostgreSQL connection URL:
```
Is that more clear?

Reply:
Yes, that is probably what it is supposed to say 👍